### PR TITLE
Update chat links

### PR DESCRIPTION
**Pull request by tarilabs**
As per https://github.com/kiegroup/drools-website/pull/138

BEFORE(today):
image

AFTER:
image

### CODE REVIEW

File: community/getHelp.html.haml [hunk 1]
```diff
@@ -86,30 +86,20 @@ title: Get help
...
 
-      Got questions? Want to discuss an issue in real-time? Then join us on IRC.
+      To follow our development discussions, join us on https://kie.zulipchat.com[Zulip chat].
 
-      Most of our project discussion take place on IRC, usually during European and American daytime.
+      - If you're working on a contribution, come talk to us on `#jbpm` stream.
+      The core developers are there during working hours (depending on their timezone).
 
-      === Connection info
+      Most of our project discussion take place on real-time chat, usually during European and American daytime.
```
Comment (MarianMacik) [inline community/getHelp.html.haml:96]: Just a nit-picking, should be `takes place`.

Comment (tarilabs) [reply community/getHelp.html.haml:96]: Fair point this sentence was originally there just moved, but then I realised it's discussion missing the s, discussions.

**Comment by tarilabs** [comment]
@krisv can you merge with squash please, before this get out of sync with master again? thank you.

**Comment by krisv** [comment]
I committed something similar already a week ago (sorry, I missed there was a similar PR for this already if there was). https://github.com/kiegroup/jbpm-website/commit/b5c12b14b785ad7ab0a2011fb2adff906c4765cb#diff-f8bdf5fc3e1014c89e1e0e739a045082R12

I did move zulip to the top of the page, as I would like to promote is as the first option.

Not sure if you think that what is there now is sufficient or you would like to improve it still?

**Comment by tarilabs** [comment]
Oh I see @krisv ; sorry didn't realised the modification are such to bump Zulip as the first medium of interaction for help. For me is a bit strange, since asynchronous communication (StackOverflow, forum) for getting help should trump synchronous one (chat) but it's fine if jBPM prefer this order!

I did raise this PR 10 days ago trying to align Drools and jBPM website, taking also some inspiration from the OptaPlanner website.

Up to you.

**Comment by krisv** [comment]
Thanks, I think I'll keep it as is for now.

**Pull request closed without merging**